src/Templates/ModelsTemplate.cshtml

```diff
@@ -99,6 +99,18 @@ func (m *Marker) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
 }
 </text>
 }
+// concatenates a slice of const values with the specified separator between each item
```

Comment:
Can't we use https://github.com/Azure/go-autorest/blob/3b13154dcaec43ea581ea56cbf6645e5bcbe8be0/autorest/utility.go#L161  ? (otherwise in some other utility package )

Reply:
No, for v3 codegen we abandoned go-autorest.

Reply:
ah so there is no utility package we can centralize stuff like this ? we will just generate everything in each file ?